time stamp
uint8 working_state
uint32 num_of_cones
float32 speed
float32 angle